client/core/core_test.go

```diff
@@ -83,8 +83,15 @@ var (
 	tUnparseableHost           = string([]byte{0x7f})
 	tSwapFeesPaid       uint64 = 500
 	tRedemptionFeesPaid uint64 = 350
-	tLogger                    = dex.StdOutLogger("TCORE", dex.LevelTrace)
+	tLogger                    = dex.StdOutLogger("TCORE", dex.LevelCritical)
```

Comment:
Wow really quieting down Core?  All the logs are oppressive, but when I bother to look at them I often see issues we were sorta ignoring.

Reply:
Whoops. I meant to undo this.